gatherers/offline.js

```diff
@@ -49,9 +48,41 @@ class Offline extends Gather {
     });
   }
 
+  static pollForOfflineResponseStatus(driver, retryCount) {
+    return driver.sendCommand('Runtime.evaluate', {
+      expression: 'window._offlineRequestStatus'
+    }).then(r => {
+      if (r.result.type !== 'undefined' || retryCount > 9) {
+        return r;
+      }
+
+      // Wait for 1.5 seconds and retry
+      return new Promise((res, _) => {
+        setTimeout(_ => {
```

Comment:
Hmm given this runs in a node and brower context the use of window will be problematic